Install/update_functions.sh

```diff
@@ -12,7 +12,7 @@ configure_line_follower(){
     delete_file $PIHOME/Desktop/line_follower_calibration.desktop
     sudo cp $PIHOME/$DEXTER/DI_Sensors/Python/di_sensors/line_follower_calibration/line_follower_calibration.desktop $PIHOME/Desktop/
     sudo chmod +x $PIHOME/Desktop/line_follower_calibration.desktop
-    sudo chmod +x $PIHOME/Dexter/DI_Sensors/Python/di_sensors/line_sensor_calibration_gui.py
+    sudo chmod +x $PIHOME/Dexter/DI_Sensors/Python/di_sensors/line_follower_calibration/line_sensor_calibration_gui.py
```

Comment:
This doesn't look so right. I'm pondering the idea of moving this code one directory up. The `di_sensors` package is more agnostic to the platform than the desktop-related things.

Reply:
Not sure I'm following. The line calibration is also completely agnostic.  And I didn't want the files for the line calibration to "pollute" the main di_sensors.

anyway, this little change in here is for a file that was already in place. The calibration GUI was already in this spot.

Reply:
I get that the file was placed in the `di_sensors` directory, initially, but even now when it's placed in its special directory, shouldn't we take this directory and move it side-by-side with the `di_sensors` directory? Like, place it in `DI_Sensors/Python`. That's what I'm thinking.

Reply:
File it as a separate issue then. We can take a look later.